src/tests/unit/utils/YotiCallbackUtils.ts

```diff
@@ -376,6 +376,35 @@ export function getDrivingPermitFields(): any {
 	return documentFields;
 }
 
+export function getDrivingPermitFieldsWithoutPostalAddress(): any {
```

Comment:
The object this returns does have a postal address, maybe we call this something like `getDrivingPermitFieldsWithoutFormattedAddress`?